website/components/content/article.md

```diff
@@ -70,3 +70,43 @@ q.page['example'] = ui.tall_article_preview_card(
 :::tip
 If you specify the `name` attribute, the whole card becomes clickable what can be used for leading into the detail view.
 :::
+
+## Wide article preview
+
+Horizontal variant.
+
+Check the full API at [ui.wide_article_preview_card](/docs/api/ui#wide_article_preview_card).
+
+```py
+    persona_pic = 'https://images.pexels.com/photos/415829/pexels-photo-415829.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=100&w=100'
+    q.page['example'] = ui.wide_article_preview_card(
```

Comment:
Look at how other examples are aligned. There is no need to start with a tab. For verification purposes, you can
```sh
cd tools/showcase
make generate
```
^^ This will generate all the screenshots and fails in case of error in python example code. To then view it in action:
```sh
cd website
npm start
```
This will start a dev mode for the docs where you can see if the example is rendered according to the expectations and is well-aligned with the rest.

Reply:
I had to use Playwright 1.17 because 1.10 is not working on macOS Monterey. I got an error which also happens on `master` branch.

![image](https://user-images.githubusercontent.com/15820796/147258707-fffee35f-8392-46af-8dcf-c993bff25528.png)

Reply:
Thanks for the info, will update the playwright version then.